src/beautifiers/r-beautifier/index.coffee

```diff
@@ -0,0 +1,24 @@
+###
+Requires [formatR](https://github.com/yihui/formatR)
+###
+path = require("path")
+
+"use strict"
+Beautifier = require('../beautifier')
+
+module.exports = class R extends Beautifier
+  name: "R beautifier"
+  link: "https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/r-beautifier/r-beautifier.r"
```

Comment:
The `link` is meant to provide reference to where the user should go in case they need to install something, such as R or `formatR` in this case. I wonder if it should be changed to https://github.com/yihui/formatR instead.